test/e2e/data/infrastructure-docker/main/clusterclass-quick-start-runtimesdk.yaml

```diff
@@ -32,6 +32,19 @@ spec:
             apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
             kind: DockerMachineTemplate
             name: quick-start-default-worker-machinetemplate
+    machinePools:
+    - class: default-worker
+      template:
+        bootstrap:
+          ref:
+            apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+            kind: KubeadmConfigTemplate
+            name: quick-start-default-worker-bootstraptemplate
```

Comment:
Just fyi, here it's fine. In this case KubeadmConfigTemplate is identical for MD/MP